minio/minio
# Added a start script to inspect command output

## Description
Using this script, post decrypt we should be able to bring up the MinIO instance with same configuration.

## Motivation and Context
Using an inspect output, we should be able to bring up MinIO instance with same configuration. This helps to debug any
issues in the setup

## How to test this PR?
Follow the below steps to verify the changes
1. `mc support inspect ALIAS/BUCKET/** --airgap`
2. Decrypt the generated inspect.enc
3. Run the command `./start-minio.sh`

Verify that MinIO instance comes up properly and we are able to create alias and access the details from bucket.
```
$ mc alias set myminio http://localhost:9000 minio minio123
$ ms ls myminio
```

## Types of changes

## Checklist:
